dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasResource.java

```diff
@@ -100,7 +80,7 @@ private long getWorkflowId(long workflowVersionId) {
         }
 
         long workflowVersionId = workflowVersion.getId();
-        long workflowId = getWorkflowId(workflowVersionId);
+        long workflowId = AliasHelper.getWorkflowId(workflowDAO, workflowVersionId);
         Workflow workflow = workflowDAO.findById(workflowId);
         workflowResource.checkEntry(workflow);
```

Comment:
This is redundant. AliasHelper.getWorkflowId already throws an exception if id is null

Reply:
Removed redundant checkEntry